src/ctim/schemas/vocabularies.cljc

```diff
@@ -255,6 +255,8 @@
     "imei"
     "imsi"
     "amp_computer_guid"
+    "ms_machine_id"
+    "s1_agent_id"
```

Comment:
Could you add a comment with a description or a reference link? What is a `ms_machine_id`? `s1_agent_id`?

Reply:
Thank you - I don't know much about these beyond what I've managed to determine by examining the examples in the issue. 

Here are my notes:
> Microsoft Windows Defender machine IDs appear to be 40 character hexadecimal strings
> Sentinel One Agent IDs appear to be 18 character decimal strings

As I'm far from an authoritative source, would it make sense to add these as a comment to `vocabularies.cljc`?

Reply:
Will do!